src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -586,10 +585,8 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
         encoding = ctcCoder.eip155TxData.encode(block.txData as EIP155TxData)
       } else if (block.sequencerTxType === TxType.EthSign) {
         encoding = ctcCoder.ethSignTxData.encode(block.txData as EthSignTxData)
-      } else if (block.sequencerTxType === TxType.createEOA) {
-        encoding = ctcCoder.createEOATxData.encode(
-          block.txData as CreateEOATxData
-        )
+      } else {
+        throw new Error(`Trying to build batch with unknown type ${block.sequencerTxType}`)
```

Comment:
This should eventually pass through the data with the unknown enum prefixing the data

Reply:
Yeah tbh the encoding should probably be handled at Geth's RPC endpoint, not in the batch submitter.